repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/core/powerfilter/badfilter/http-logging.cfg.xml

```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I've made a separate PR at: https://github.com/rackerlabs/repose/pull/1317

I won't remove the commit from this branch unless you want me to. When the other is merged, Git should figure things out, right?